source/default/_patterns/02-molecules/vue-widget/src/vue-clock/vue-clock.vue

```diff
@@ -34,31 +34,34 @@
   </div>
 </template>
 
-<script>
-export default {
+<script lang="ts">
```

Comment:
Any reason we converted this entire example over instead of creating a new parallel example with TS?

Reply:
My thinking is that we have several other single file component Vue examples that could be indicative of how to write a Vue SFC in JavaScript. I am currently writing up documentation as well of how to convert a Vue SFC to TypeScript.